src/views/List.jsx

```diff
@@ -51,6 +67,7 @@ export function List({ data, listToken }) {
 					})}
 				</ul>
 			</form>
+			{sortErrorMessage && <p> {sortErrorMessage} </p>}
```

Comment:
I love error handling!
I think for this to truly work we would need to write this conditionally, as in:

`sortErrorMessage ? {<p> There was an error sorting your list </p>} : <the entire search form & entire listItem form>`

Reply:
I think this also works. For this case, if sortErrorMessage is truthy, render the paragraph.